FastTimeSeries/src/models/StackedLSTM.jl

```diff
@@ -0,0 +1,50 @@
+# StackedLSTM.jl
+#
+# Layers for stacked LSTM (referenced from https://github.com/sdobber/FluxArchitectures.jl)
+
+mutable struct StackedLSTMCell{A}
+	chain::A
+end
+
+"""
+	StackedLSTM(in, out, hiddensize, layers)
+
+Stacked LSTM network. Feeds the data through a chain of LSTM layers, where the hidden state
+of the previous layer gets fed to the next one. The first layer corresponds to
+`LSTM(in, hiddensize)`, the hidden layers to `LSTM(hiddensize, hiddensize)`, and the final
+layer to `LSTM(hiddensize, out)`. Takes the keyword argument `init` for the initialization
+of the layers.
+
+"""
+function StackedLSTM(in::Int, out::Integer, hiddensize::Integer, layers::Integer;
+			init=Flux.glorot_uniform)
+	if layers == 1
+		chain = Chain(LSTM(in, out; init=init))
+	elseif layers == 2
+		chain = Chain(LSTM(in, hiddensize; init=init),
+					  LSTM(hiddensize, out; init=init))
+	else
+		chain_vec = [LSTM(in, hiddensize; init=init)]
+		for i = 1:layers - 2
+			push!(chain_vec, LSTM(hiddensize, hiddensize; init=init))
+		end
+		chain = Chain(chain_vec..., LSTM(hiddensize, out; init=init))
+	end
+	return StackedLSTMCell(chain)
+end
```

Comment:
Something is a little weird here: you're returning a `Cell`, but inside that cell are `Recur`s and not `LSTMCells`. Ideally `StackedLSTMCell` would be structured like the other Flux RNN cells, but if that's not possible I'd recommend renaming to `StackedLSTM` to better represent the model as stateful + containing internal mutation (transitively via `Recur` from `LSTM`).